### ISSUE TITLE
New feature request: Image regression model

### ISSUE THREAD

**Issue by curtfehr** [dotnet/machinelearning#4680]
It would be nice to have a regression model that works on images, so an image rank can be predicted.  eg. Train images of the sky with a cloudiness rank of 1-100, get prediction as a float instead of bucketing into classifications.

**Comment by yaeldekel** [comment]
Hi @curtfehr ,
By "regression model", do you mean a DNN model?
ML.NET has some non-DNN regression algorithms, you could try using them, perhaps with some featurization such as [ApproximatedKernelMapping](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Transforms/KernelCatalog.cs#L40)

**Comment by curtfehr** [comment]
Thanks I'll give that a try. 

Is there any plans to add more scenarios to the "Add Machine Learning" GUI? I'm finding that very useful to quickly spin up POC's and test ideas without having to spend much time in the code.

**Comment by yaeldekel** [comment]
Yes, I believe the plan is to continue developing Model Builder and add more scenarios to it. Thank you for the feedback!
